maintainers/maintainer-list.nix

```diff
@@ -2983,6 +2983,12 @@
     githubId = 57923898;
     name = "Elyhaka";
   };
+  em0lar = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.